### ISSUE TITLE
Keyboard navigable video "bars" (transcript, related resources, ...)

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/ocw-hugo-themes#1126]
As a user navigating with a keyboard, I'd like to be able to expand/collapse the tabs below videos (e.g., transcripts).

#### Acceptance Criteria:

- [x] Using keyboard navigation, be able to focus and manipulate (expand/collapse) video bars (like transcripts, related resources).
    - also the download button, and any other buttons within the bars.
- [x] focus should be visibly indicated *It currently is not on the iframe itself, even though that is focusable*.

### Comments
- It's probably enough to use more semantic elements for the chevron, etc. Right now they are `<i>`, `<span>`,... but should probably be buttons; ditto with the download button, which is currently just a div.
- playwright tests for the associated functionality using accessible locators (like `page.getByRole(role, { name: "Transcript" })` would be a good way to cement this. (The inability to use `getByRole` in a playwright test is how this issue was discovered.)

**Comment by ibrahimjaved12** [comment]
I'll have more questions to this,
but for now, I think we will also want to look into what kind of behaviour we want to have for embed videos with regards to keyboard navigation. Or, do we want to leave them as they are?

Here is an example:
https://ocw.mit.edu/courses/18-02sc-multivariable-calculus-fall-2010/pages/1.-vectors-and-matrices/part-a-vectors-determinants-and-planes/session-7-cross-products/

<img width="875" alt="image" src="https://user-images.githubusercontent.com/109785089/234560640-4cc05f59-8d22-4bce-96ba-4e4dde356cf0.png">

So i think for this case, we will want to have two kinds of behaviours for keyboard navigation: 
a) to be able to navigate to the video page (this one's functionality is slightly different from the one mentioned in the issue, since others (transcript/related resources) expand collapsed bars, but here they should allow users to redirect to the video page)
b) download button for embed videos should still be keyboard-functional as mentioned in the issue.

**Comment by ChristopherChudzicki** [comment]
@ibrahimjaved12 I think we will pretty much get the behavior we want by using more semantically meaningful elements. Probably....
```
view video page              ... <a></a>
download button              ... <button></button>
transcripts toggle           ... <button></button>
related resources toggle     ... <button></button>
```

**Comment by ibrahimjaved12** [comment]
> @ibrahimjaved12 I think we will pretty much get the behavior we want by using more semantically meaningful elements. Probably....
> 
> ```
> view video page              ... <a></a>
> download button              ... <button></button>
> transcripts toggle           ... <button></button>
> related resources toggle     ... <button></button>
> ```

Fair enough. Makes sense
